modules/friendlyshared.js

```diff
@@ -144,7 +150,7 @@ Twinkle.shared.callbacks = {
 			return;
 		}
 
-		Morebits.status.info( 'Info', 'Will add the shared IP address template to the top of the user\'s talk page.' );
+		Morebits.status.info( $.i18n._('tw-core-morebits-info'), $.i18n._('tw-sharedip-will-add') );
 		text += params.value + '|' + params.organization;
 		if( params.value === 'Shared IP edu' && params.contact !== '') {
```

Comment:
Yes, but what is the alternative? To move all callbacks into the l10n file? I am not saying that is doable. I am saying that will require a LOT of time, and not something I can do alone.